### DISCUSSION TITLE
Cannot import() a bundled JS

### DISCUSSION THREAD

**Discussion by yu0A** [webpack/webpack#17375] [Q&A] [answered]
# Bug report

**What is the current behavior?**
Compiled with problems:X

ERROR

[eslint] 
C:\xxx\src\views\svg\js\svg.min.js
  12:22347  error  Unexpected empty method 'addEventListener'                       @typescript-eslint/no-empty-function
  12:22751  error  Unexpected empty method 'removeEventListener'                    @typescript-eslint/no-empty-function
  12:22767  error  Unexpected empty function 'Ot'                                   @typescript-eslint/no-empty-function
  12:26038  error  Expected a conditional expression and instead saw an assignment  no-cond-assign
  12:53130  error  Expected a conditional expression and instead saw an assignment  no-cond-assign
  12:61796  error  Unexpected empty method 'clearTransformsFromQueue'               @typescript-eslint/no-empty-function
  12:75980  error  Expected a conditional expression and instead saw an assignment  no-cond-assign

✖ 7 problems (7 errors, 0 warnings)
**If the current behavior is a bug, please provide the steps to reproduce.**
1. get svg.min.js from https://cdnjs.com/libraries/svg.js
2. put svg.min.js into "./js/svg.min.js"
3. import ("./js/svg.min.js") in business JS code

**What is the expected behavior?**
import SVG, which works like: var svg = SVG('#my-svg');

**Other relevant information:**
webpack version:  5.75.0
Node.js version: v16.19.1
Operating System: Windows 10
Additional tools:

**Comment by alexander-akait** [comment, marked as answer]
Not related to `webpack`, you have `eslint` errors for `typescript` code, fix them

**Reply by yu0A** [reply]
Yes, those "errors" are not errors. Thanks for your answer.
